# Array of Detection3D messages

Header 				header					# Standard message header.
Detection3D[] 		detections				# A list of the detected objects.
